molo/core/middleware.py

```diff
@@ -135,3 +139,17 @@ def process_response(self, request, response):
                 site_settings.global_ga_tracking_code, request, response)
 
         return response
+
+
+class MultiSiteRedirectToHomepage(object):
+
+    def process_request(self, request):
+        if request.path.startswith('/admin/pages/'):
```

Comment:
The issue wasn't this, I didn't actually fix it, I as fixing other tests and those passed as a result. Not sure what made it pass